### ISSUE TITLE
Elixir templates in foo.html.eex form use the HTML icon instead of EEx

### ISSUE THREAD

**Issue by nickjj** [oivva/st-boxy#136]
#### General info

I am pretty sure the EEx icon is the 4 yellow line icon right? When the file is `foo.eex` I correctly see that. I also correctly see the Elixir icon for `foo.ex` files.

#### Expected Behavior

Given a file name of `foo.html.eex` I would expect to see an EEx icon in the sidebar.

#### Current Behavior

Instead, `foo.html.eex` renders an HTML icon (the green `<>` icon) in the sidebar.

#### Steps to Reproduce (for bugs)

1. Create `foo.html.eex`
2. Direct eyeballs to sidebar
3. Stare in dismay at the HTML icon

#### Your Environment
```
- Boxy Theme: 4.0.0
- Sublime Text: 3126
- Platform: linux
- Package Control: True
- Dependencies:
    * mdpopups: 1.10.0
    * markdown: 2.6.6
    * pygments: 2.1a0
    * jinja2: 2.8
```

**Comment by oivva** [comment]
Thanks, I'll add it in the next release :wink:

**Comment by nickjj** [comment]
Excellent, great job on this theme btw.

**Comment by oivva** [comment]
It will be fixed by #138.
